src/utils/types.zig

```diff
@@ -433,6 +443,69 @@ pub fn containsPointer(comptime T: type) ?bool {
     };
 }
 
+/// This is an exact copy of std.mem.eqlBytes because it is private
```

Comment:
just a side note, this is another example of why i push for all our methods to be public

Reply:
It's easy to copy and paste the code. It's just a question of whose responsibility it is to maintain compatibility.